services/tenant-ui/frontend/src/components/innkeeper/tenants/deleteTenant/DeleteTenant.vue

```diff
@@ -0,0 +1,53 @@
+<template>
+  <div>
+    <Button
+      :title="$t('tenants.settings.deleteTenant')"
+      icon="pi pi-trash"
+      class="p-button-rounded p-button-icon-only p-button-text"
+      @click="openModal"
+    />
+    <Dialog
+      v-model:visible="displayModal"
+      :style="{ minWidth: '500px' }"
+      :header="'Delete Tenant'"
+      :modal="true"
+      @update:visible="handleClose"
+    >
+      <ConfirmTenantDeletion
+        :tenant="props.tenant"
+        @success="$emit('success')"
+        @closed="handleClose"
+      />
+    </Dialog>
+  </div>
+</template>
+
+<script setup lang="ts">
+// Types
+import { TenantRecord } from '@/types/acapyApi/acapyInterface';
```

Comment:
I think we should simplify the import categories here. Like you did in the last PR.